plugins/generator-1.17.1/forge-1.17.1/mappings/entities.yaml

```diff
@@ -0,0 +1,478 @@
+  - sheep
+EntityShoulderRiding: ShoulderRidingEntity
+EntityShulker:
+  - Shulker
+  - EntityType.SHULKER
+  - shulker
+EntityShulkerBullet:
+  - ShulkerBullet
+  - EntityType.SHULKER_BULLET
+  - shulker_bullet
+EntitySilverfish:
+  - Silverfish
+  - EntityType.SILVERFISH
+  - silverfish
+EntitySkeleton:
+  - Skeleton
+  - EntityType.SKELETON
+  - skeleton
+EntitySkeletonHorse:
+  - SkeletonHorse
+  - EntityType.SKELETON_HORSE
+  - skeleton_horse
+EntitySlime:
+  - Slime
+  - EntityType.SLIME
+  - slime
+EntitySmallFireball:
+  - SmallFireball
+  - EntityType.SMALL_FIREBALL
+  - small_fireball
+EntitySnowman:
+  - SnowGolem
+  - EntityType.SNOW_GOLEM
+  - snow_golem
+EntitySnowball:
+  - Snowball
+  - EntityType.SNOWBALL
+  - snowball
+EntitySpectralArrow:
+  - SpectralArrow
+  - EntityType.SPECTRAL_ARROW
+  - spectral_arrow
+EntitySpellcasterIllager: SpellcasterIllager
+EntitySpider:
+  - Spider
+  - EntityType.SPIDER
+  - spider
+EntitySquid:
+  - Squid
+  - EntityType.SQUID
+  - squid
+EntityStray:
+  - Stray
+  - EntityType.STRAY
+  - stray
+EntityTraderLlama:
+  - TraderLlama
+  - EntityType.TRADER_LLAMA
+  - trader_llama
+EntityTameable: TamableAnimal
+EntityThrowable: ThrowableProjectile
+EntityTropicalFish:
+  - TropicalFish
+  - EntityType.TROPICAL_FISH
+  - tropical_fish
+EntityTurtle:
+  - Turtle
+  - EntityType.TURTLE
+  - turtle
+EntityEgg:
+  - ThrownEgg
+  - EntityType.EGG
+  - egg
+EntityEnderPearl:
+  - ThrownEnderpearl
+  - EntityType.ENDER_PEARL
+  - ender_pearl
+EntityExpBottle:
+  - ThrownExperienceBottle
+  - EntityType.EXPERIENCE_BOTTLE
+  - experience_bottle
+EntityPotion:
+  - ThrownPotion
+  - EntityType.POTION
+  - potion
+EntityTrident:
+  - ThrownTrident
+  - EntityType.TRIDENT
+  - trident
+EntityVex:
+  - Vex
+  - EntityType.VEX
+  - vex
+EntityVillager:
+  - Villager
+  - EntityType.VILLAGER
+  - villager
+EntityVindicator:
+  - Vindicator
+  - EntityType.VINDICATOR
+  - vindicator
+EntityWaterMob: WaterAnimal
+EntityWanderingTrader:
+  - WanderingTrader
+  - EntityType.WANDERING_TRADER
+  - wandering_trader
+EntityWitch:
+  - Witch
+  - EntityType.WITCH
+  - witch
+EntityWither:
+  - WitherBoss
+  - EntityType.WITHER
+  - wither
+EntityWitherSkeleton:
+  - WitherSkeleton
+  - EntityType.WITHER_SKELETON
+  - wither_skeleton
+EntityWitherSkull:
+  - WitherSkull
+  - EntityType.WITHER_SKULL
+  - wither_skull
+EntityWolf:
+  - Wolf
+  - EntityType.WOLF
+  - wolf
+EntityZombie:
+  - Zombie
+  - EntityType.ZOMBIE
+  - zombie
+EntityZombieHorse:
+  - ZombieHorse
+  - EntityType.ZOMBIE_HORSE
+  - zombie_horse
+EntityZombieVillager:
+  - ZombieVillager
+  - EntityType.ZOMBIE_VILLAGER
+  - zombie_villager
+EntityPhantom:
+  - Phantom
+  - EntityType.PHANTOM
+  - phantom
+EntityRavager:
+  - Ravager
+  - EntityType.RAVAGER
+  - ravager
+EntityLightningBolt:
+  - LightningBolt
+  - EntityType.LIGHTNING_BOLT
+  - lightning_bolt
+EntityPlayer:
+  - Player
+  - EntityType.PLAYER
+  - player
+EntityPlayerMP:
+  - ServerPlayer

```

Comment:
I couldn't find this one in the class list, could you recheck if it still exists?

Reply:
It is ServerPlayer now